packages/storybook/.storybook/main.js

```diff
@@ -24,10 +24,11 @@ module.exports = {
     '@storybook/addon-a11y/register',
   ],
   typescript: {
-    check: __DEV__, // Netlify is breaking the deploy with this settings on. So deactivate on release
+    check:  __DEV__, // Netlify is breaking the deploy with this settings on. So deactivate on release
```

Comment:
AFAIK, this should be formated by prettier but isn't.

@eps1lon By default, we ignore dot files/folders, shouldn't set [`dot: true`](https://github.com/isaacs/node-glob#dots) in the [glob options](https://github.com/mui-org/material-ui/blob/cb6b69baa828623393b1ed0ade8efde309ac06b0/scripts/prettier.js#L33)?

Reply:
For prettier that makes sense, yes.

Reply:
Ok, I will add it in the next batch. It should be a one-line change. Then, once committed we can benefit from it on material-ui-x.

Reply:
> It should be a one-line change.

Those are usually the kind of changes the lead to days of work. At least I thought so in https://github.com/mui-org/material-ui/pull/22814: Just bump the browser version and done :laughing: